src/Bicep.Core.Samples/Files/Parameters_LF/main.diagnostics.bicep

```diff
@@ -53,6 +53,7 @@ param myArrayParam array = [
 
 // alternative array parameter
 param myAlternativeArrayParam array {
+//@[36:107) [BCP153 (Warning)] Parameter modifiers are deprecated and will be removed in a future release. Use decorators instead (see https://aka.ms/BicepSpecParams for examples). |{\n  default: [\n    'a'\n    'b'\n    'c'\n    newGuid()\n    utcNow()\n  ]\n}|
```

Comment:
>https://aka.ms/BicepSpecParams [](start = 135, length = 30)

Does the entire team have access to modify this aka.ms link?

Reply:
I tried to set the Project Bicep team as the owner, but it didn't work, so I ended up using the unicomb DL.

Reply:
Yeah, that works.